packages/jsapi-utils/src/TableUtils.test.ts

```diff
@@ -175,6 +190,30 @@ describe('quick filter tests', () => {
     expect(result).toBe(expectedResult);
   }
 
+  function testFilterWithType(
+    functionName: string,
+    text,
+    expectedFn,
+    type,
+    ...args
+  ) {
+    const column = makeFilterColumn(type);
+    const filter = column.filter();
+
+    const expectedResult = makeFilterCondition();
+
+    filter[expectedFn].mockReturnValueOnce(expectedResult);
+
+    const result = TableUtils[functionName](column, text);
+
+    if (args.length > 0) {
+      expect(filter[expectedFn]).toHaveBeenCalledWith(...args);
+    } else {
+      expect(filter[expectedFn]).toHaveBeenCalled();
+    }
```

Comment:
Shouldn't need to split this case up. `...args` will just be an empty array, and calling `.toHaveBeenCalledWith(...args)` just means it will have been called with no arguments, which is actually a stricter check than just `.toHaveBeenCalled()` (which would pass whether it was called with or without arguments).